src/containers/Tenant/Diagnostics/TopicData/columns/Columns.scss

```diff
@@ -0,0 +1,18 @@
+.ydb-diagnostics-topic-data-columns {
+    &__lags-header {
```

Comment:
It's not used

Reply:
it is used here https://github.com/ydb-platform/ydb-embedded-ui/blob/7cefb6d025c84cc3669cfa7704f7aaa199f6c077/src/containers/Tenant/Diagnostics/Partitions/columns/columns.tsx#L97